### ISSUE TITLE
model evaluation method

### ISSUE THREAD

**Issue by Crankyxx** [abi/screenshot-to-code#268]
How to evaluate the performance of the model on generalized data, such as comparing the original screenshots with the generated results? Are there any indicators?

**Comment by abi** [comment]
Yes, see https://github.com/abi/screenshot-to-code/blob/main/Evaluation.md and https://github.com/abi/screenshot-to-code/blob/main/blog/evaluating-claude.md

What are you looking to do? Would love contributions back to the repo,
